### DISCUSSION TITLE
Welcome to SYT-Web-Redesign Discussions!

### DISCUSSION THREAD

**Discussion by JimmyTron** [SpaceyaTech/SYT-Web-Redesign#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hamisirizwan** [comment]
Hello Team.
Thank you for your valuable contributions to the SpaceYaTech Website Re-Design Open-source project. To streamline our collaboration and ensure efficient code review and progress, We would like to invite you to fill this form So we can better organize weekly standups to keep everyone updated as well as share tasks to be achieved.

https://forms.gle/k1d5WU563zYFMma6A

@JimmyTron @jumalaw98 @sonylomo @Ase020 @Rioba-Ian @Hudson-obai
